ingress/controllers/nginx/controller.go

```diff
@@ -739,19 +740,38 @@ func (lbc *loadBalancerController) getUpstreamServers(ngxCfg config.Configuratio
 				server = servers[defServerName]
 			}
 
+			if rule.HTTP == nil && host != defServerName {
+				glog.Info("multiple root ISSUE")
```

Comment:
maybe send an event, and turn the message into a WARNING or ERROR? Please either extend the message itself or add a comment.